pkg/cmd/repo/autolink/list/http.go

```diff
@@ -0,0 +1,56 @@
+package list
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/cli/cli/v2/api"
+	"github.com/cli/cli/v2/internal/ghinstance"
+	"github.com/cli/cli/v2/internal/ghrepo"
+)
+
+type AutolinkGetter struct {
+	HttpClient *http.Client
+}
+
+func NewAutolinkGetter(httpClient *http.Client) *AutolinkGetter {
+	return &AutolinkGetter{
+		HttpClient: httpClient,
+	}
+}
+
+func (a *AutolinkGetter) Get(repo ghrepo.Interface) ([]autolink, error) {
```

Comment:
**Suggestion**

I suspect this might be better as `List` rather than `Get` (and same for the struct type). I think it better implies the intention, particularly at the call site in the command.

Reply:
I like that. Especially since if we continue to copy this pattern throughout the command set the naming for each command's http interface will be consistent 👍